modules/reactor/doc/content/source/meshgenerators/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator.md

```diff
@@ -0,0 +1,40 @@
+# HexagonConcentricCircleAdaptiveBoundaryMeshGenerator
+
+!syntax description /Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator
+
+## Overview
+
+The `HexagonConcentricCircleAdaptiveBoundaryMeshGenerator` object generates a 2D mesh for concentric circles with a hexagon enclosure with or without external ducts. It is similar to [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) and the majority of the parameters and functionalities of this object are described in the documentation of [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md). The major unique feature of this object is that the `num_sectors_per_side` parameter will be overridden to enable stitching to another hexagonal mesh (provided via `input` to this object). This object therefore creates hexagonal meshes that can be directly stitched with hexagonal meshes generated by [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) and [`PatternedHexMeshGenerator`](/PatternedHexMeshGenerator.md). This object only works for hexagons instead of general polygons.
```

Comment:
```suggestion
The `HexagonConcentricCircleAdaptiveBoundaryMeshGenerator` object generates a 2D mesh for concentric circles with a hexagon enclosure with or without external ducts. It is similar to [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) and the majority of the parameters and functionalities of this object are described in the documentation of [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md). The major unique feature of this object is that the [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/num_sectors_per_side) parameter will be overridden to enable stitching to another hexagonal mesh (provided via [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/input) to this object). This object therefore creates hexagonal meshes that can be directly stitched with hexagonal meshes generated by [`PolygonConcentricCircleMeshGenerator`](/PolygonConcentricCircleMeshGenerator.md) and [`PatternedHexMeshGenerator`](/PatternedHexMeshGenerator.md). This object only works for hexagons instead of general polygons.
```